tests/manage/monitoring/conftest.py

```diff
@@ -188,7 +188,7 @@ def stop_mon():
         if (
             not split_index == 1
             or config.ENV_DATA["platform"].lower()
-            in constants.MANAGED_SERVICE_PLATFORMS
+            not in constants.MANAGED_SERVICE_PLATFORMS
```

Comment:
This change makes the first condition pointless for all non-mananged deployments.

Reply:
Right, thanks

Reply:
Updated